File: app/src/main/java/com/moh/mhealth/SimpleLocation.java
package com.moh.mhealth;

import android.location.Location;

public class SimpleLocation {
    private double latitude;
    private double longitude;

    public SimpleLocation(Location loc) {
        this.latitude = loc.getLatitude();
        this.longitude = loc.getLongitude();
    }

    public SimpleLocation(double latitude, double longitude) {
        this.latitude = latitude;
        this.longitude = longitude;
    }

    public double getLatitude() {
        return this.latitude;
    }

    public double getLongitude() {
        return this.longitude;
    }

}

File: app/src/main/java/com/moh/mhealth/objects/SimpleLocation.kt
package com.moh.mhealth.objects

import android.location.Location

class SimpleLocation {
    var latitude: Double
        private set
    var longitude: Double
        private set

    constructor(loc: Location) {
        latitude = loc.latitude
        longitude = loc.longitude
    }

    constructor(latitude: Double, longitude: Double) {
        this.latitude = latitude
        this.longitude = longitude
    }
}